### PR TITLE
re-organize chain abstraction

### PR DESCRIPTION
**Pull request by gagdiez**
"Learn" now reads closer to a blog-post (I think this is the general direction we will want from now on for Learn), while "Build" has all the technical details, while still explaining enough of the concepts

### CODE REVIEW

File: docs/1.concepts/abstraction/chain-signatures.md [hunk 1]
```diff
@@ -4,54 +4,57 @@ title: Chain Signatures
...
 
-1. A [smart contract](../basics/accounts/smartcontract.md) that holds requests for multi-chain signatures.
-2. A [multiparty computation](https://www.zellic.io/blog/mpc-from-scratch/) service listening for signature requests.
-3. A multi-chain [relayer](./relayers.md), which can submit signed transactions to other networks.
-
-:::info
-This section presents an overview of Chain Signatures. To create one, please switch to the [**building a Chain Signature**](../../8.abstraction/chain-signatures.md) document.
+:::info Searching for code?
+This document presents only broad concepts, to include Chain Signatures in your project check the [**build documentation**](../../8.abstraction/chain-signatures.md)
 :::
```
Comment (bucanero) [inline docs/1.concepts/abstraction/chain-signatures.md:13]:
```suggestion
:::info Searching for code?

This document presents only broad concepts, to include Chain Signatures in your project check the [**build documentation**](../../8.abstraction/chain-signatures.md)

:::
```

File: docs/1.concepts/abstraction/chain-signatures.md [hunk 2]
```diff
@@ -4,54 +4,57 @@ title: Chain Signatures
...
-There are four steps involved on Chain Signatures:
+Controlling accounts on other chains is made possible thanks to the interaction between three elements:
 
-1. [Create a Payload](#1-create-a-payload) - The user creates the transaction / message they want to sign
-2. [Signature Request](#2-request-signature) - The user calls the NEAR `multichain` contract, requesting to sign the transaction
-3. [MPC Signing Service](#3-sign-with-mpc) - A service captures the call, and returns the signed the transaction for the user
-4. [Relay Signed Payload](#4-relaying-the-signature) - The signed payload is then sent to the destination chain for execution.
+1. A deterministic way to derive [**multiple foreign addresses**](#one-account-multiple-chains) from one NEAR account
+2. A **smart contract** that receives requests to sign transaction for other chains
+3. A [**multiparty computation**](#3-mpc-signing-service) service that provides the smart contract with the signatures
```
Comment (bucanero) [inline docs/1.concepts/abstraction/chain-signatures.md:23]:
```suggestion
3. A [**multiparty computation service**](#3-mpc-signing-service) that provides the signatures to the smart contract
```

File: docs/1.concepts/abstraction/chain-signatures.md [hunk 3]
```diff
@@ -4,54 +4,57 @@ title: Chain Signatures
...
+<hr class="subsection" />
 
-```rust
-  pub fn sign(payload: [u8; 32], path: String) -> Signature
-```
-_[See the full code in Github](https://github.com/near/mpc-recovery/blob/bc85d66833ffa8537ec61d0b22cd5aa96fbe3197/contract/src/lib.rs#L263)_
+### A Contract for Chain Signatures
+A deployed smart contract is used to request signatures for transactions on other blockchains.
+
+The contract has a method called `sign` that takes two parameters:
```
Comment (bucanero) [inline docs/1.concepts/abstraction/chain-signatures.md:49]:
```suggestion
The contract has a `sign` method that takes two parameters:
```

File: docs/1.concepts/abstraction/chain-signatures.md [hunk 4]
```diff
@@ -4,54 +4,57 @@ title: Chain Signatures
...
+  1. The `payload` (transaction) to be signed for the target blockchain
+  2. The `path` that identifies the account we want to use to sign the payload.
```
Comment (bucanero) [inline docs/1.concepts/abstraction/chain-signatures.md:51]:
```suggestion
  2. The `path` that identifies the account you want to use to sign the payload.
```

File: docs/1.concepts/abstraction/chain-signatures.md [hunk 5]
```diff
@@ -62,49 +65,38 @@ Note that each call will take one block, and thus result on ~1s of waiting. Afte
...
 
-<details>
-<summary> What is an MPC Service? </summary>
-
-MPC (multi-party computation) allows independent actors to do shared computations on private information, without revealing secrets to each-other.
+<hr class="subsection" />
 
-NEAR uses its own MPC service to safely sign transactions for other chains on behalf of the user. In practice, **no single node** on the MPC can **sign by itself** since they do **not hold the user's keys**. Instead, nodes create signature-shares which are aggregated through multiple rounds to jointly sign the payload.
+### Multi-Party Computation Service (MPC)
+Multi-party computation (MPC) allows independent actors to do shared computations on private information, without revealing secrets to each-other.
```
Comment (bucanero) [inline docs/1.concepts/abstraction/chain-signatures.md:75]:
```suggestion
Multi-party computation (MPC) allows independent actors to do shared computations on private information without revealing secrets to each other.
```

File: docs/8.abstraction/chain-signatures.md [hunk 6]
```diff
@@ -7,104 +7,160 @@ import Tabs from '@theme/Tabs';
...
 
-Chain Signatures unlock the ability for a single account to transact across multiple blockchain protocols, giving ownership of cross-chain accounts, data, and assets to one NEAR account.
+NEAR accounts can safely control accounts across different blockchain protocols, allowing each NEAR account to have true ownership of cross-chain data and assets.
 
-In this document we cover the steps necessary to sign a transaction for another chain, we recommend you to read our [**overview of Chain Signatures**](../1.concepts/abstraction/chain-signatures.md) before, as well as some of its [**use cases**](../1.concepts/abstraction/signatures/use-case.md).
+![chain-signatures](/docs/assets/welcome-pages/chain-signatures-overview.png)
+_Diagram of a chain signature in NEAR_
 
-:::info
+There are five steps involved on Chain Signatures:
```
Comment (bucanero) [inline docs/8.abstraction/chain-signatures.md:15]:
```suggestion
There are five steps involved in Chain Signatures:
```

File: docs/8.abstraction/chain-signatures.md [hunk 7]
```diff
@@ -7,104 +7,160 @@ import Tabs from '@theme/Tabs';
...
 
-See our [web-app example](https://github.com/near-examples/near-multichain) and [component example](https://test.near.social/md1.testnet/widget/chainsig-sign-eth-tx) showing how a NEAR account can create an Ethereum testnet transaction.
+1. [Deriving the Foreign Address](#1-deriving-foreigner-the-address) - Reconstruct the address that will be controlled on the target blockchain
+2. [Creating a Payload](#2-creating-the-payload) - Create the transaction / message that we want to sign for the target blockchain
+3. [Requesting a Signature](#3-requesting-the-signature) - Call the NEAR multichain contract, requesting to sign the payload
+4. [Reconstructing the Signature](#4-reconstructing-the-signature) - Reconstruct the signature from the MPC service's response
+5. [Relaying the Signed Payload](#5-relaying-the-signature) - The signed payload is then sent to the destination chain for execution.
 
+:::info Looking for examples?
+See our [web-app example](https://github.com/near-examples/near-multichain) and [component example](https://test.near.social/md1.testnet/widget/chainsig-sign-eth-tx) showing how a NEAR account can create transactions on Ethereum.
```
Comment (bucanero) [inline docs/8.abstraction/chain-signatures.md:24]: @think-in-universe posted that there's a bug on the BOS component, and he made a fork with the fix:
https://test.near.social/bot.testnet/widget/chainsig-sign-eth-tx

[Reference](https://pagodaplatform.slack.com/archives/C03PKHB82QP/p1709914663561569?thread_ts=1709603754.834769&cid=C03PKHB82QP)

File: docs/8.abstraction/chain-signatures.md [hunk 8]
```diff
@@ -7,104 +7,160 @@ import Tabs from '@theme/Tabs';
...
 
 ---
 
-## Create the Payload
-The first step is to use Chain Signatures is to construct a payload (transaction, message, data, etc.) for the target blockchain platform. This variates depending on the target blockchain, but in general, it's a hash of the message or transaction to be signed.
+## 1. Deriving the Foreign Address
+
+Chain Signatures use [`derivation paths`](../1.concepts/abstraction/chain-signatures.md#one-account-multiple-chains) to represent accounts on the target blockchain.
+
+The external address can be deterministically derived from the NEAR's address (e.g. `ana.near`), a derivation path (a string such as `ethereum-1`, `ethereum-2`, etc), and the MPC service's public key.
```
Comment (bucanero) [inline docs/8.abstraction/chain-signatures.md:33]:
```suggestion
The external address can be deterministically derived from the NEAR address (e.g., `ana.near`), a derivation path (a string such as `ethereum-1`, `ethereum-2`, etc), and the MPC service's public key.
```

**Review by bucanero — APPROVED** [review]

File: docs/8.abstraction/chain-signatures.md (hunk 7 above)
Comment (gagdiez) [reply docs/8.abstraction/chain-signatures.md]: Agree, lets use it

File: docs/8.abstraction/chain-signatures.md [hunk 9]
```diff
@@ -1,110 +1,181 @@
...
-See our [web-app example](https://github.com/near-examples/near-multichain) and [component example](https://test.near.social/md1.testnet/widget/chainsig-sign-eth-tx) showing how a NEAR account can create an Ethereum testnet transaction.
+This guide will take you through a step by step process for creating a Chain Signature.
 
-::: 
+⭐️ For a deep dive into the concepts of Chain Signatures see [What are Chain Signatures?](/concepts/abstraction/chain-signatures)
+
+⭐️ For complete examples of a NEAR account performing Eth transactions:
+
+- [web-app example](https://github.com/near-examples/near-multichain)
+- [component example](https://test.near.social/md1.testnet/widget/chainsig-sign-eth-tx)
```
Comment (gagdiez) [inline docs/8.abstraction/chain-signatures.md:23]:
```suggestion
- [component example](https://test.near.social/bot.testnet/widget/chainsig-sign-eth-tx) 
```

File: docs/8.abstraction/chain-signatures.md [hunk 10]
```diff
@@ -1,110 +1,181 @@
...
+:::info
 
-</TabItem>
+⭐️ For a deep dive into the concepts of Chain Signatures see [What are Chain Signatures?](/concepts/abstraction/chain-signatures)
 
-</Tabs>
+⭐️ For complete examples of a NEAR account performing Eth transactions:
+
+- [web-app example](https://github.com/near-examples/near-multichain)
+- [component example](https://test.near.social/md1.testnet/widget/chainsig-sign-eth-tx)
```
Comment (gagdiez) [inline docs/8.abstraction/chain-signatures.md:179]:
```suggestion
- [component example](https://test.near.social/bot.testnet/widget/chainsig-sign-eth-tx) 
```

**Pull request merged**